internal/storage/dynamodb.go

```diff
@@ -0,0 +1,191 @@
+package storage
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"log"
```

Comment:
Can we use the updated slog?

Reply:
Sure i'll wait for @UjwalaK to merge the logger MR and update to use it.

Reply:
Fixed